compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -2318,7 +2377,11 @@ public void visit(BLangRecordLiteral recordLiteral) {
 
                     if (names.contains(name)) {
                         this.dlog.error(keyExpr.pos, DiagnosticCode.DUPLICATE_KEY_IN_RECORD_LITERAL,
-                                        recordLiteral.expectedType.getKind().typeName(), name);
+                                        recordLiteral.expectedType.getKind().typeName(),
+                                        name);
```

Comment:
Can't these continue to be
```suggestion
                                        recordLiteral.expectedType.getKind().typeName(), name);
```